.github/workflows/ci_check_pr.yaml

```diff
@@ -1,131 +0,0 @@
-name: CI Checks on PRs
```

Comment:
@leomayleomay @srid we can't get rid of this file, we run E2E tests with newman.

Reply:
@Datron it's my bad, I thought Srid meant this file redundant, I believe he meant my changes to this file redundant